src/state/trajectory/logics.ts

```diff
@@ -334,21 +343,96 @@ const loadFileViaUrl = createLogic({
 });
 
 const fileConversionLogic = createLogic({
-    process(deps: ReduxLogicDeps, dispatch) {
+    process(deps: ReduxLogicDeps, dispatch, done) {
         dispatch(
             setStatus({
                 status: VIEWER_IMPORTING,
             })
         );
+        done();
     },
     type: CONVERT_FILE,
 });
 
+const setConversionEngineLogic = createLogic({
+    async process(deps: ReduxLogicDeps): Promise<{
+        engineType: AvailableEngines;
+        template: Template;
+        templateMap: TemplateMap;
+    }> {
+        const {
+            httpClient,
+            action,
+            uiTemplateUrlRoot,
+            uiBaseTypes,
+            uiCustomTypes,
+            uiTemplateDownloadUrlRoot,
+        } = deps;
+        const baseTypes = await httpClient
+            .get(`${uiTemplateDownloadUrlRoot}/${uiBaseTypes}`)
+            .then((baseTypesReturn: AxiosResponse) => {
+                console.log(baseTypesReturn);
```

Comment:
are these console logs needed?

Reply:
the ones we added no, removing them now. there is one console log from a different commit in a catch block that i will leave because it seems like it might be important.